MonoGame.Framework/GamePlatform.cs

```diff
@@ -16,6 +16,7 @@ abstract partial class GamePlatform : IDisposable
 
         protected TimeSpan _inactiveSleepTime = TimeSpan.FromMilliseconds(20.0);
         protected bool _needsToResetElapsedTime = false;
+        bool ignoredInitialActivation = false;

```

Comment:
Is it better to have an `enum ActiveState { NotInitialized, Active, Inactive }` instead of these two booleans?
Also, the PR title really seems to be "Fix Game.IsActive notification on initial window creation" correct? Not really that the value is incorrect...

Reply:
> Is it better to have an `enum ActiveState { NotInitialized, Active, Inactive }` instead of these two booleans?

No.  That would bloat the code base.

My concern: 
Is there need for this change?  

In the cases that you describe, the user is fully aware the game may be starting without them.

This is an edge case that the only time that should be a problem is a game without a menu or title screen at startup. 

As this is an additional 32 bits of memory, and a one time event dispatch, I am not opposed to the change, I just question the need.

Reply:
> > Is it better to have an `enum ActiveState { NotInitialized, Active, Inactive }` instead of these two booleans?
> 
> No. That would bloat the code base.
> 
> My concern: Is there need for this change?
> 
> In the cases that you describe, the user is fully aware the game may be starting without them.
> 
> This is an edge case that the only time that should be a problem is a game without a menu or title screen at startup.
> 
> As this is an additional 32 bits of memory, and a one time event dispatch, I am not opposed to the change, I just question the need.

I definitely see a need for this change. The problem isn't really that the game already started.
Here is a contrived example: I want to start a game but I know that it takes 30 seconds to load up. So I start it but switch to my browser to do something else for those 30 seconds. If the game is done loading and is showing the title screen (which is hidden by the browser), you click buttons in the browser and without knowing it you also click buttons in the game.